src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -75,14 +100,17 @@ A mutable structure that stores information about a randomized linear solver's b
 - `iterations::Int64`, the number of iterations of the solver.
 - `converged::Bool`, a flag to indicate whether the system has converged by some measure.
 - `true_res`, a boolean indicating if we want the true residual computed instead of approximate.
-- `dist_info::DistInfo`, a structure that stores information about the sub-Exponential distribution.
+- `dist_info::SEDistInfo`, a structure that stores information about the sub-Exponential distribution.
+  includes the parameter `sigma2`, which is the variance parameter of a sub-Exponential distribution,
+  and the parameter `omega`. This values have been approximated for many sampling methods; but the 
+  user can specify there own, if desired. The parameter `eta` can also be used to scale the default
+  parameters by having the variance be set to `sigma2/eta` and the other parameter be `omega/eta`.
```

Comment:
Reference the docstring for `SEDistInfo`

Reply:
I have added this docstring